pycbc/events/stat.py

```diff
@@ -132,6 +132,25 @@ def coinc(self, s0, s1, slide, step): # pylint:disable=unused-argument
         """
         return (s0**2. + s1**2.) ** 0.5
 
+    def coinc_multiifo(self, slide, step,
```

Comment:
I think this looks ok for a first draft. I don't think we want to hold up merging this, but it might be good for @tdent and @spxiwh to have a look and think if this is the right interface for most kinds of things we'd want. I.E. should there be a way to get information by IFO and apply ranking in a non-ifo symmetric way? I think that work can probably be orthogonal to @tapaimarton's effort to get the coincidence / background / workflow working.